tests/update/patch_sle.pm

```diff
@@ -59,6 +59,7 @@ sub patching_sle {
         # Update origin system on zVM that is controlled by autoyast profile and reboot is done by end of autoyast installation
         # So we skip reboot here after fully patched on zVM to reduce times of reconnection to s390x
         if (!get_var('UPGRADE_ON_ZVM')) {
+            select_console 'root-console';
```

Comment:
i suggest some comments needed here why add this ?

Reply:
LGTM

Reply:
Updated, thanks.